morphir/runtime/src/org/finos/morphir/runtime/TestSummary.scala

```diff
@@ -9,7 +9,8 @@ private[runtime] case class CoverageInfo(
 case class TestSummary(
     report: String,
     countsByModule: Map[(PackageName, ModuleName), TestResultCounts],
-    private val coverage: CoverageInfo
+    private val coverage: CoverageInfo,
+    private val individualTestResults: Map[FQName, String]
```

Comment:
This is better, but I don't like it just being to `String` - that makes it difficult to tell what the results were in a programatic way. Why can't this be `SingleTestResult`?